### PR TITLE
[PBA-3406] Set the AppID to something publicly avavailable.

### PR DESCRIPTION
**Pull request by jonslenkooyala**

### CODE REVIEW

File: ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
 @implementation OOCastManagerFetcher
 +(OOCastManager*) fetchCastManager {
-  return [OOCastManager getCastManagerWithAppID:@"D4C7BD4A" namespace:@"urn:x-cast:ooyala"];
+  return [OOCastManager getCastManagerWithAppID:@"B3560E04" namespace:@"urn:x-cast:ooyala"];
```
Comment (mityaha) [inline ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m]: Should it be default receiver, not custom one?

Comment (jonslenkooyala) [inline ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m]: B3560E04 is the default receiver. Do you mean it is better for it to be the custom one? Thanks.

Comment (mityaha) [inline ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m]: It was staging custom receiver, you replaced it by product default receiver.  Shouldn't it be product custom receiver 4172C76F ?

Comment (mityaha) [inline ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m]: I just think that as it was custom receiver, then let's keep custom receiver.  Custom receiver, unlike default one, allows controlling v3 version.  Besides, custom receiver allows user to modify receiver code, whereas default receiver allows only to customize icons.  We actually may discontinue support for default receiver altogether at some point.

Comment (jonslenkooyala) [inline ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m]: I'll update it to be custom then, thanks.

**Comment by mityaha** [comment]
+1

**Pull request merged**